pkg/storegateway/chunk_bytes_pool.go

```diff
@@ -39,12 +34,14 @@ func (p *chunkBytesPool) Get(sz int) (*[]byte, error) {
 		return buffer, err
 	}
 
-	p.requestedBytes.Add(float64(sz))
-	p.returnedBytes.Add(float64(cap(*buffer)))
+	p.poolByteStats.WithLabelValues("Get", "Requested").Add(float64(sz))
+	p.poolByteStats.WithLabelValues("Get", "Cap").Add(float64(cap(*buffer)))
 
 	return buffer, err
 }
 
 func (p *chunkBytesPool) Put(b *[]byte) {
+	p.poolByteStats.WithLabelValues("Put", "Len").Add(float64(len(*b)))
```

Comment:
I am curious, how would the `Len` stats be useful when returning stuff to the pool?

Reply:
I can lowercase it.

The len is showing how much we really used from the cap.